worlds/cvcotm/docs/en_Castlevania - Circle of the Moon.md

```diff
@@ -0,0 +1,116 @@
+# Castlevania: Circle of the Moon
+
+## Quick Links
+- [Setup](/tutorial/Castlevania%20-%20Circle%20of%20the%20Moon/setup/en)
+- [Options Page](/games/Castlevania%20-%20Circle%20of%20the%20Moon/player-options)
+- [PopTracker Pack](https://github.com/sassyvania/Circle-of-the-Moon-Rando-AP-Map-Tracker-/) (The Releases are out-of-date. Click the green "Code" button and
+then "Download ZIP" for the most up-to-date version.)
+- [Repo of the original non-Archipelago version of this randomizer](https://github.com/calm-palm/cotm-randomizer)
+- [Web version of the above randomizer](https://rando.circleofthemoon.com/)
+
+## What does randomization do to this game?
+
+Almost all items that you would normally find on pedestals throughout the game have had their locations changed. In addition to
+magic items (barring the Dash Boots which you always start with) and stat max ups, the DSS Cards have been added to the
+item pool as well; you will now receive these as randomized items rather than by farming them via enemy drops.
+
+## Can I use any of the alternate classes?
+
+Yes. All alternate classes (Magician, Fighter, Shooter, and Thief) are all unlocked and usable from the start by registering
+their names shown on the Data Select screen.
+
+If you intend to play as the Magician, it is highly recommended to put all of your cards in `start_inventory_from_pool` so that
```

Comment:
~~I did not know putting them in one of the start inventories doesn't account them in logic from the start. Ooops!~~ That sentence is being reworded anyway because I fixed the issue of cards you already have not spawning on the game's side.